cmd/config/config.go

```diff
@@ -101,3 +105,12 @@ func (c *UpfConfig) Validate() error {
 func (c *UpfConfig) Unmarshal() error {
 	return v.UnmarshalExact(c)
 }
+
+var configFileAvailable bool
+
+func SetWatchConfig(onConfigChange func(e fsnotify.Event)) {
```

Comment:
Two verbs in name is too much. `WatchConfig`?

Reply:
\+